### ISSUE TITLE
ConvertFrom-MySQLiteDB doesn't restore blobs

### ISSUE THREAD

**Issue by jdhitsolutions** [jdhitsolutions/MySQLite#15]
Blobbed properties via clixml data don't get properly restored.

```powershell
PS C:\> [pscustomobject]@{Name="Jeff";Date=(Get-date);Msg = "This isn't that difficult. Is it?";Bits = (Get-Service bits)} | ConvertTo-MySQLiteDB -Path d:\temp\tst3.db -TableName test
PS C:\> Convertfrom-mySQLitedb -Path D:\temp\tst4.db -PropertyTable propertymap_System_Management_Automation_PSCustomObject -TableName test | fl


Name : Jeff
Date : 9/12/2022 9:01:31 AM
Msg  : This isn't that difficult. Is it?
Bits :
```

The `Bits` property is null. There is data in the table.

```powershell
PS C:\> Invoke-MySQLiteQuery -Path D:\temp\tst4.db -q "Select * from test"

Name Date                 Msg                               Bits
---- ----                 ---                               ----
Jeff 9/12/2022 9:01:31 AM This isn't that difficult. Is it? {60, 79, 98, 106...}
```

**Comment by jdhitsolutions** [comment]
Data exists, just not getting properly restored.

```powershell
PS C:\> convertfrom-mySQLitedb -Path D:\temp\tst4.db -TableName test -RawObject

Name Date                 Msg                               Bits
---- ----                 ---                               ----
Jeff 9/12/2022 9:01:31 AM This isn't that difficult. Is it? {60, 79, 98, 106...}
```

**Comment by jdhitsolutions** [comment]
This appears to be fixed with the release of v0.10.0
